File: 백준/Bronze/10813. 공 바꾸기/공 바꾸기.c
#include <stdio.h>
int main() {
	int n, m, i, j,tmp;
	int arr[101];
	for (int k = 0; k < 101; k++) {
		arr[k] = k;
	}
	scanf("%d %d", &n, &m);
	for (int k = 1;  k <= m; k++) {
		scanf("%d %d", &i, &j);
		tmp = arr[i];
		arr[i] = arr[j];
		arr[j] = tmp;
	}

	for (int k = 1; k <= n; k++)
		printf("%d ", arr[k]);

	return 0;
}

File: 백준/Bronze/10813. 공 바꾸기/공 바꾸기.java
import java.util.Scanner;

public class Main {
	public static void main(String[]args) {
		Scanner sc = new Scanner(System.in);
		int N = sc.nextInt();
		int M = sc.nextInt();
		int arr[]= new int[N];
		int tmp;
		for(int k=0; k<N; k++)arr[k]=k+1;
		
		for(int k =0; k<M; k++) {
			int i = sc.nextInt();
			int j = sc.nextInt();
			tmp = arr[j-1];
			arr[j-1]=arr[i-1];
			arr[i-1]=tmp;
		}
		for(int k=0; k<N; k++) {
			System.out.print(arr[k]+" ");
		}
		
	}
}